UM/Mesh/MeshData.py

```diff
@@ -228,6 +228,15 @@ def reserveFaceCount(self, num_faces):
 
         self._vertex_count = 0
         self._face_count = 0
+
+
+    def setIndices(self, indices):
+        self._indices = indices
+        self._face_count = int(self._indices.size / 3)
+
+    def setVertices(self, vertices):
+        self._vertices = vertices
+        self._vertex_count = int(self._vertices.size / 3)
```

Comment:
I think we should. 
We have a matrix that looks like this: [[x0, y0, z0], [x1, y1, z0],..., [xn, yn, zn]]
The size then reports n * 3. In order to get n we need to divide by three.